driver-tests/osgi/src/test/java/com/datastax/driver/osgi/BundleOptions.java

```diff
@@ -48,6 +49,19 @@ public static MavenArtifactProvisionOption guavaBundle() {
         return mavenBundle("com.google.guava", "guava", "16.0.1");
     }
 
+    public static CompositeOption lz4Bundle() {
+        return new CompositeOption() {
+
+            @Override
+            public Option[] getOptions() {
+                return options(
+                        systemProperty("cassandra.compression").value(ProtocolOptions.Compression.LZ4.name()),
```

Comment:
Because this system property is not always present, subsequent tests would fail with the native container (runs all tests in the same JVM), while they succeed with the forked one.